integration/integration_suite_test.go

```diff
@@ -1,16 +1,90 @@
 package integration_test
 
 import (
+	"context"
+	"fmt"
+	"github.com/labstack/echo/v4"
+	. "github.com/onsi/gomega"
+	"github.com/tidepool-org/clinic/api"
+	integrationTest "github.com/tidepool-org/clinic/integration/test"
 	dbTest "github.com/tidepool-org/clinic/store/test"
 	"github.com/tidepool-org/clinic/test"
+	xealthTest "github.com/tidepool-org/clinic/xealth/test"
+	"go.uber.org/fx"
+	"net/http/httptest"
+	"os"
+	"sync"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 )
 
+var app *fx.App
+var server *echo.Echo
+var shorelineStub *httptest.Server
+var seagullStub *httptest.Server
+var authStub *httptest.Server
+var xealthStub *xealthTest.XealthServer
+
 func TestSuite(t *testing.T) {
 	test.Test(t)
 }
 
-var _ = BeforeSuite(dbTest.SetupDatabase)
-var _ = AfterSuite(dbTest.TeardownDatabase)
+var _ = BeforeSuite(setupEnvironment)
+var _ = AfterSuite(teardownEnvironment)
+
+func setupEnvironment() {
+	dbTest.SetupDatabase()
+
+	authStub = integrationTest.AuthStub()
+	seagullStub = integrationTest.SeagullStub()
+	shorelineStub = integrationTest.ShorelineStub()
+	xealthStub = xealthTest.ServerStub()
+	keycloakStub := integrationTest.KeycloakStub()
+
+	Expect(os.Setenv("LOG_LEVEL", "error")).To(Succeed())
```

Comment:
I suspect you'd be better off using [testing.T.Setenv](https://cs.opensource.google/go/go/+/refs/tags/go1.21.1:src/testing/testing.go;l=1421) here..?

I don't know if ginkgo supports it via GinkgoT() or not though..?

But it's useful because it handles cleanup of the variables after the test for you.

Reply:
TIL this exists, thanks!

Reply:
> TIL this exists, thanks!

It's not super old (go 1.17).